moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -303,6 +303,8 @@ bool MoveItConfigData::outputCHOMPPlanningYAML(const std::string& file_path)
   emitter << YAML::Key << "collision_clearence" << YAML::Value << "0.2";
   emitter << YAML::Key << "collision_threshold" << YAML::Value << "0.07";
   emitter << YAML::Key << "use_stochastic_descent" << YAML::Value << "true";
+  emitter << YAML::Key << "enable_failure_recovery" << YAML::Value << "false";
```

Comment:
Maybe set to `true` by default, since CHOMP isn't particularly robust.

Reply:
sure will do this.